README.md

```diff
@@ -85,66 +39,70 @@ to hide your IP while using FreeTube.
+Go to [FreeTube's Documentation](https://docs.freetubeapp.io/) if you'd like to know more about how to operate FreeTube and its features.
 
-## Download Links
-### Official Downloads
-* [GitHub Releases](https://github.com/FreeTubeApp/FreeTube/releases)
+## Screenshots
+<img alt="subscriptions" src="https://user-images.githubusercontent.com/78101139/209993178-ebec42bc-7082-42c7-b283-3073e39e938c.png" width=300> <img alt="trending videos" src="https://user-images.githubusercontent.com/78101139/209993253-29519061-570a-4640-aee3-a38f8e9e8ae5.png" width=300> <img alt="watching video" src="https://user-images.githubusercontent.com/78101139/209993912-c82ca1f6-b8f8-4d6b-a67f-3d5a36686484.png" width=300> <img  alt="video comments" src="https://user-images.githubusercontent.com/78101139/209994008-26ffb0cb-e2da-4a6c-9d0b-0c438d3dc41c.png" width=300> 
 
-* [FreeTube Website](https://freetubeapp.io/#download)
+## How to install
 
-* Flatpak on Flathub: [Download](https://flathub.org/apps/details/io.freetubeapp.FreeTube) [Source](https://github.com/flathub/io.freetubeapp.FreeTube)
+There is a deployed PWA (progressive web app) available here:
 
-#### Automated Builds (Nightly / Weekly)
-Builds are automatically created from changes to our development branch via [GitHub Actions](https://github.com/FreeTubeApp/FreeTube/actions?query=workflow%3ABuild).
+[https://marmadilemanteater.github.io/freetube/#/](https://marmadilemanteater.github.io/freetube/#/)
 
-The first build with a green check mark is the latest build.  You will need to have a GitHub account to download these builds.
+There is an APK which can be downloaded here:
 
-### Unofficial Downloads
-These builds are maintained by the community.  While they should be safe, download at your own risk.  There may be issues with using these versus the official builds.  Any issues specific with these builds should be sent to their respective maintainer.
+[https://github.com/MarmadileManteater/FreeTubeCordova/releases](https://github.com/MarmadileManteater/FreeTubeCordova/releases)
 
-* Arch User Repository (AUR): [Download](https://aur.archlinux.org/packages/freetube-bin/)
+## How to build and test
 
-* Chocolatey: [Download](https://chocolatey.org/packages/freetube/)
+```bash
+    npm run pack:browser
+    # This creates the cordova project, 
+    # copies the dist folder, browserifies 
+    # it, and replaces a bunch of functions
+    # with cordova equivalents.
+    # After your first build, most of the
+    # build components will be recycled
+    # from that build.
+    npm run build:cordova
+    # This opens up the cordova application 
+    # in a web browser
+    npm run run:cordova browser
+    # This opens up the cordova application
+    # on an android device connected with 
+    # debugging enabled
+    npm run run:cordova android
```

Comment:
When I wrote this, I was using `npm`, but upstream uses `yarn` and I also use `yarn` now, so it might make sense to update this to the yarn commands.
```suggestion
    yarn run:cordova android
```